docs/source/scenarios/container_inventory_management.rst

```diff
@@ -483,41 +483,41 @@ Random Repositioning
      - Resource Shortage
      - Repositioning Number
    * - global_trade.22p_l0.0
-     - 2,240,000
-     - 1,010,009 :math:`\pm` 20,942
-     - 548,240 :math:`\pm`  14,600
+     - 896,000
+     - 191,342 :math:`\pm` 14,520
+     - 1,198,423 :math:`\pm`  47,658
    * - global_trade.22p_l0.1
-     - 2,240,000
-     - 1,027,395 :math:`\pm` 19,183
-     - 188,160 :math:`\pm`  12,940
+     - 896,000
+     - 132,728 :math:`\pm` 10,871
+     - 767,837 :math:`\pm`  54,388
    * - global_trade.22p_l0.2
-     - 2,240,000
-     - 1,035,851 :math:`\pm`  4,352
-     - 181,240 :math:`\pm`   5,240
+     - 896,000
+     - 132,728 :math:`\pm`  10,871
+     - 767,837 :math:`\pm`   54,388
    * - global_trade.22p_l0.3
-     - 2,239,460
-     - 1,032,480 :math:`\pm`  1,332
-     - 190,220 :math:`\pm`   8,920
+     - 895,460
+     - 156,253 :math:`\pm`  13,413
+     - 815,633 :math:`\pm`  41,173
    * - global_trade.22p_l0.4
-     - 2,244,068
-     - 1,034,412 :math:`\pm` 11,689
-     - 186,080 :math:`\pm`   6,280
+     - 900,068
+     - 159,324 :math:`\pm` 16,897
+     - 805,260 :math:`\pm` 63235
    * - global_trade.22p_l0.5
-     - 2,244,068
-     - 1,042,869 :math:`\pm` 16,146
-     - 188,720 :math:`\pm`   7,880
+     - 900,068
+     - 155,491 :math:`\pm` 15,008
+     - 803,360 :math:`\pm` 45,249
    * - global_trade.22p_l0.6
-     - 2,244,068
-     - 1,096,502 :math:`\pm` 26,896
-     - 302,280 :math:`\pm`  27,540
+     - 900,068
+     - 189,518 :math:`\pm` 10,296
+     - 877,715 :math:`\pm`  45,844
    * - global_trade.22p_l0.7
-     - 2,244,068
-     - 1,144,981 :math:`\pm`  5,355
-     - 283,520 :math:`\pm`  25,700
+     - 900,068
+     - 218,620 :math:`\pm`  8,695
+     - 755,990 :math:`\pm`  35800
    * - global_trade.22p_l0.8
-     - 2,241,716
-     - 1,154,184 :math:`\pm`  7,043
-     - 270,960 :math:`\pm`   2,240
+     - 897,716
+     - 216,480 :math:`\pm`  13,768
+     - 743,825 :math:`\pm`   41,461
 
 Quick Start
```

Comment:
@ysqyang @wesley-stone please also add the experiment result for MDQN and GraphA2C, and also add the example link near to the experiment result table.

Reply:
Currently there is no GraphA2C implementation in this repo. Maybe MDQN is enough.

Reply:
I have added GNN results.